### PR TITLE
140 multimedia configuration

### PR DESCRIPTION
**Pull request by newtewt**
#140 

@garethbowen  I've added docs on how to config and supported types. 

- What encodings are recommended, eg: better compression rates, better hardware acceleration for video decoding, etc.
I selected the major use(h264, jpeg, mp3) options as our recommendations. They're widely supported and compressed. 

- Recommendations for file sizes, particularly for video.
- Recommendations for image and video dimensions (eg: there's no point having lots of pixels and then scaling it down to fit on the phone screen).

I wasn't sure what we should add for these. As a project could potentially have wildly different hardware depending on use. I did add a note about keeping in mind that lower end devices have limited storage and screen densities.

### CODE REVIEW

File: configuration/forms.md [hunk 1]
```diff
@@ -565,3 +565,107 @@ If using a [XLSForm](http://xlsform.org/) and using [`medic-conf`](https://githu
 | `start` | A timestamp of when the form entry was started. |
 | `end` | A timestamp of when the form entry ended. |
 | `today` | Day of the survey. |
+
+
+# Multimedia in Forms
+
+## Multimedia Formats
+
+There are many supported formats for video, audio, and images. We recommend using h.264(mpeg) for video, jpeg for images, and mp3 for audio. When creating videos or images keep in mind the dimensions and storage capabilities on phones that may be used. Lower end phones have smaller storage and screen sizes.
```
Comment (garethbowen) [inline configuration/forms.md]: I would mention something at the start of this paragraph about _how_ the media is rendered, eg: "When rendering images, video, and audio the CHT uses the browser's built in rendering tools. This means you can render any media format that is supported by the [minimum version of Chrome](https://github.com/medic/medic-docs/blob/master/installation/supported-software.md#dependencies).".

File: configuration/forms.md [hunk 2]
```diff
@@ -565,3 +565,107 @@ If using a [XLSForm](http://xlsform.org/) and using [`medic-conf`](https://githu
...
+
+
+## Configuration
+
+To play multimedia from forms you need to add elements to your xml and upload the corresponding multimedia to couchdb as an attachment to your form.  
+
+### Form Config
+  Add an xml element of text and another element of value. Set form equal to the type of multimedia being used(video, audio, image). The value element must contain `jr://file_name.suffix` where `file_name.suffix` is the name of your multimedia file uploaded to couchdb. 
+
+  EX:
```
Comment (garethbowen) [inline configuration/forms.md]: Nit: Use the full word "Example:" here.

File: configuration/forms.md [hunk 3]
```diff
@@ -565,3 +565,107 @@ If using a [XLSForm](http://xlsform.org/) and using [`medic-conf`](https://githu
...
+### Form Config
+  Add an xml element of text and another element of value. Set form equal to the type of multimedia being used(video, audio, image). The value element must contain `jr://file_name.suffix` where `file_name.suffix` is the name of your multimedia file uploaded to couchdb. 
+
+  EX: 
+  ```
+  <text id="somevideo">
+	  <value form="video">jr://video.mp4</value>
+</text>
+  ```
+  Display EX:
```
Comment (garethbowen) [inline configuration/forms.md]: Nit: here too.

**Comment by newtewt** [comment]
@garethbowen  updated with changes.

**Pull request merged**